src/Homepage/Homepage.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react'
+import styled from 'styled-components'
```

Comment:
You could also use `'styled-components/macro'`here, as you did in your global styles.